src/App.tsx

```diff
@@ -1,47 +1,55 @@
 import React from 'react'
+import moment from 'moment'
 import logo from './logo.svg'
 import './App.css'
 import FileImportButton from './Components/FileImportButton'
 import TextField from '@material-ui/core/TextField'
+import Grid from '@material-ui/core/Grid'
 import xml2js from 'xml2js'
-import Attempt from './types'
+import { Attempt } from './types'
+import DurationLineChart from './Components/DurationLineChart'
 
 const App = () => {
   const [attemptHistory, setAttemptHistory] = React.useState<Attempt[]>([])
+  const [errorMessage, setErrorMessage] = React.useState<string>()
 
   function handleFile(e: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
Not setting mandatory return types?

Reply:
Adding typing to the component definitions